arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -534,7 +534,7 @@ RocksDBEdgeIndex::RocksDBEdgeIndex(TRI_idx_iid_t iid,
         RocksDBIndex::ESTIMATOR_SIZE);
     TRI_ASSERT(_estimator != nullptr);
   }
-  TRI_ASSERT(iid != 0);
+  TRI_ASSERT(iid == 1 || iid == 2);
```

Comment:
will this work after an upgrade of 3.3 to devel? will edge indexes created in 3.3 keep their original ids and make the assertion fail?

Reply:
rocksdb always creates edge indexes with ID 1 and 2